modules/nf-core/star/genomegenerate/tests/main.nf.test

```diff
@@ -0,0 +1,40 @@
+nextflow_process {
+
+    name "Test Process STAR_GENOMEGENERATE"
+    script "../main.nf"
+    process "STAR_GENOMEGENERATE"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "star"
+    tag "star/genomegenerate"
+
+    test("homo_sapiens") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test_fasta' ],
+                    [file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)]
+                ])
+                input[1] = Channel.of([
+                    [ id:'test_gtf' ],
+                    [file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)]
+                ])
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert path(process.out.index.get(0).get(1)).exists() },
+                { assert path(process.out.versions.get(0)).getText().contains("star") },
+                { assert path(process.out.versions.get(0)).getText().contains("samtools") },
+                { assert path(process.out.versions.get(0)).getText().contains("gawk") }
+            )
```

Comment:
👀 🙂

Reply:
Yeah, I know, I asked yesterday already, snapshot is stable but not working on conda

Reply:
ah, it's for this one I see. Sorry, didn't recognize it. Do you mind investigating this a bit more? Are there maybe different version of star etc. in conda compared to the other containers?

Reply:
okay, had a look and found I prime example of why to snapshot the version:

`environment.yml`:
```yaml
 - bioconda::star=2.7.10a
 ```

container:
```
'star=2.7.9a'
```

Reply:
Looking forward to wave

Reply:
ok, so @FriederikeHanssen is the one that created the last containers.
and she did request the right version.
cf https://github.com/BioContainers/multi-package-containers/pull/2377/files#diff-f6be21f33f5bb216e9a0baac781bc0d81da47c65d70a53701dfc6cac35c80966R263

so something is wrong on the biocontainers side

Reply:
I updated both actually with star .10 and .9 and must have used the wrong tag to update :(

Reply:
Yeah, that was probably it.
thanks for your help anyway